tests/test_person.py

```diff
@@ -43,6 +45,32 @@ def test_get_person_by_full_name(self):
             )
             person = gazu.person.get_person_by_full_name("John Did")
             self.assertEqual(person["id"], "person-2")
+        with requests_mock.mock() as mock:
+            mock.get(
+                gazu.client.get_full_url("data/persons"),
+                text=json.dumps(
+                    [
+                        {
+                            "first_name": "John",
+                            "last_name": "Doe",
+                            "id": "person-01",
+                        },
+                        {
+                            "first_name": "JohnDid",
+                            "last_name": "",
+                            "id": "person-2",
+                        },
+                        {
+                            "first_name": "Ema",
+                            "last_name": "Doe",
+                            "id": "person-3",
+                        },
+                    ]
+                ),
+            )
+            person = gazu.person.get_person_by_full_name("JohnDid")
```

Comment:
It would be more interesting to search with something like "Jhon Doe". 
NB: In some way this function is inefficient, the search should be done on the server side.